Source/RunActivity/Viewer3D/Processes/RenderProcess.cs

```diff
@@ -132,20 +140,32 @@ internal void Start()
 
             DisplaySize = new Point(GraphicsDevice.Viewport.Width, GraphicsDevice.Viewport.Height);
 
-            if (Game.Settings.ShaderModel == 0)
-                Game.Settings.ShaderModel = GraphicsDevice.GraphicsDeviceCapabilities.PixelShaderVersion.Major;
-            else if (Game.Settings.ShaderModel < 2)
-                Game.Settings.ShaderModel = 2;
-            else if (Game.Settings.ShaderModel > 3)
-                Game.Settings.ShaderModel = 3;
+            try
+            {
+                // Validate that the DirectX feature level is one we understand
+                Enum.Parse(typeof(ORTS.Settings.UserSettings.DirectXFeature), "Level" + Game.Settings.DirectXFeatureLevel);
+            }
+            catch (Exception)
+            {
+                Game.Settings.DirectXFeatureLevel = "";
+            }
```

Comment:
```suggestion
            // Validate that the DirectX feature level is one we understand
            if (!Enum.IsDefined(typeof(ORTS.Settings.UserSettings.DirectXFeature), "Level" + Game.Settings.DirectXFeatureLevel))
                Game.Settings.DirectXFeatureLevel = "";
```
Prefer Enum.IsDefined over an exception-swallowing try-catch.